packages/block-library/src/navigation/edit.js

```diff
@@ -120,34 +137,42 @@ function Navigation( {
 								? navIcons[
 										`justify${ upperFirst(
 											attributes.itemsJustification
-										) }Icon`
+										) }`
 								  ]
-								: navIcons.justifyLeftIcon
+								: navIcons.justifyLeft
 						}
 						label={ __( 'Change items justification' ) }
 						isCollapsed
 						controls={ [
 							{
-								icon: navIcons.justifyLeftIcon,
+								icon: justifyLeft,
 								title: __( 'Justify items left' ),
 								isActive:
 									'left' === attributes.itemsJustification,
 								onClick: handleItemsAlignment( 'left' ),
 							},
 							{
-								icon: navIcons.justifyCenterIcon,
+								icon: justifyCenter,
 								title: __( 'Justify items center' ),
 								isActive:
 									'center' === attributes.itemsJustification,
 								onClick: handleItemsAlignment( 'center' ),
 							},
 							{
-								icon: navIcons.justifyRightIcon,
+								icon: justifyRight,
 								title: __( 'Justify items right' ),
 								isActive:
 									'right' === attributes.itemsJustification,
 								onClick: handleItemsAlignment( 'right' ),
 							},
+							{
+								icon: justifySpaceBetween,
+								title: __( 'Space between items' ),
+								isActive:
+									'spaceBetween' ===
+									attributes.itemsJustification,
```

Comment:
Continued from the [above](https://github.com/WordPress/gutenberg/pull/28169/files#diff-610616a7f92cf5567753ff6e54b42e95ffd4b9607984b72c814f56279e87fff4R68) suggestion:

```
isActive:
	'space-between' ===
	attributes.itemsJustification,
onClick: handleItemsAlignment(
	'space-between'
),
```